### DISCUSSION TITLE
Show if a comment was edited

### DISCUSSION THREAD

**Discussion by iTrooz** [Docile-Alligator/Infinity-For-Reddit#909] [Ideas]
On reddit you can see if a comment has been edited (maybe even posts ? not sure)

I'm willing to work on this and submit a PR if you give me the get-go

**Comment by Wladefant** [comment]
@iTrooz It Is definitely worth it

**Comment by Wladefant** [comment]
Let me check the JSON response first.

_Originally posted by @Docile-Alligator in https://github.com/Docile-Alligator/Infinity-For-Reddit/issues/388#issuecomment-911680604_

**Comment by Wladefant** [comment]
Please mark all your feature requests as feature requests

**Comment by iTrooz** [comment]
> Please mark all your feature requests as feature requests

Done ! I'm not sure how to add a new label to them

**Comment by Wladefant** [comment]
> > Please mark all your feature requests as feature requests
> 
> Done ! I'm not sure how to add a new label to them

Unfortunately you can't, you made the right thing

**Comment by iTrooz** [comment]
> Let me check the JSON response first.
> 
> _Originally posted by @Docile-Alligator in [#388 (comment)](https://github.com/Docile-Alligator/Infinity-For-Reddit/issues/388#issuecomment-911680604)_

I'm going to check if the API have it

Btw, what do you use to check the API ? I'm currently messing with mitmproxy + BurpSuite + HttpCanary but I feel like none of them really meet my needs

**Comment by iTrooz** [comment]
You can see from this request (taken from Infinity itself using HttpCanary) that there is indeed an "edited" key
https://paste.gg/p/anonymous/b6f56d34de624227916d77701b4873b8
Post :https://www.reddit.com/r/OutOfTheLoop/comments/2rdy77/why_does_everyone_say_edit_when_they_make_an_edit/

**Comment by Docile-Alligator** [comment]
@iTrooz Nice. That will be very helpful. Thank you.

**Comment by iTrooz** [comment]
Hey, so while implementing the feature, I found these two classes [HistoryPostRecyclerViewAdapter](https://github.com/Docile-Alligator/Infinity-For-Reddit/blob/master/app/src/main/java/ml/docilealligator/infinityforreddit/adapters/HistoryPostRecyclerViewAdapter.java) and [PostRecyclerViewAdapter](https://github.com/Docile-Alligator/Infinity-For-Reddit/blob/master/app/src/main/java/ml/docilealligator/infinityforreddit/adapters/PostRecyclerViewAdapter.java), which seems *extremely* similar

Is there any reason why the code had to be duplicated in two classes ?

It seems strange to me to have to implement the feature two times in both of these classes (it's just a few lines tho)

**Comment by saworden** [comment]
@iTrooz Did a PR get made for this? Or were you maybe waiting for an answer to that last question? It sounds like you got the feature working.

**Comment by iTrooz** [comment]
Hey ! It's a bit of both actually, I asked the question and then totally forgot about it.

I indeed made some code that works for the comments. I will submit it
As for the posts.. I can't work on it right now because the "design" view doesn't show up anymore for some reason. I'll probably submit another PR to merge the two classes I've mentioned before that anyway

EDIT: #1388

**Comment by candroid-man** [comment]
Any updates on this?

**Comment by iTrooz** [comment]
Yeah, it was added

**Reply by candroid-man** [reply]
I'm an idiot don't mind me
